rollup.config.mjs

```diff
@@ -0,0 +1,75 @@
+// rollup.config.mjs
+
+import commonjs from '@rollup/plugin-commonjs';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import terser from '@rollup/plugin-terser';
+import alias from '@rollup/plugin-alias';
+
+const banner = `/* Copyright 2015-${new Date().getFullYear()} PlayCanvas Ltd */\n`;
+
+const builds = [
+    {
+        name: 'playcanvas-spine.3.6',
+        lib: 'contrib/spine-ts/build/3.6/spine-core.js'
+    },
+    {
+        name: 'playcanvas-spine.3.8',
+        lib: 'contrib/spine-ts/build/3.8/spine-core.js'
+    },
+    // {
+    //     name: 'playcanvas-spine.4.0',
+    //     lib: 'spine40' // may require wrapper or adjustment
+    // },
```

Comment:
What's this? Are you intending on adding 4.0 support as well or something?

Reply:
It's 2 lines of code and some testing.  I am working up the courage.  

Want to make sure the build system is ready in this PR.

Reply:
OK, I'll hold off on merging for now then. Let me know when you think you're done. 😄

Reply:
Im done, not doing 4.0 in this PR.  Ready to merge.  Just wanted to make the build config clearer by showing the version progression.

Reply:
Oh OK. I'd rather not merge commented out or incomplete stuff to main though. Can you delete this for now and if you want to add in a separate PR, do it there?

Reply:
ok, just a note to others, removing now.  Keen to start on examples PR.

Reply:
completed in: 1a543df